app/src/main/java/com/nextcloud/talk/activities/CallActivity.kt

```diff
@@ -1953,7 +2024,7 @@ class CallActivity : CallBaseActivity() {
         }
     }
 
-    private fun hangup(shutDownView: Boolean) {
+    private fun hangup(shutDownView: Boolean, endCallForAll: Boolean?) {
```

Comment:
The parameter "all" is set to true only when we need to end call for everyone. Otherwise, it is not set(so passing null should be okay here)

Reply:
I think when there are only 2 possibilities we should make use of true/false, so don't make it nullable.
If the API needs null in case it's false, then implement it just before the API call.